ReSwift/CoreTypes/Middleware.swift

```diff
@@ -11,4 +11,4 @@ import Foundation
 public typealias DispatchFunction = (Action) -> Void
 public typealias GetState = () -> StateType?
 public typealias Middleware =
-    (DispatchFunction?, @escaping GetState) -> (@escaping DispatchFunction) -> DispatchFunction
+    (@escaping DispatchFunction, @escaping GetState) -> (@escaping DispatchFunction) -> DispatchFunction
```

Comment:
To the reviewers: I'm not sure how to format it best to not exceed 100 symbols limit: 

```
public typealias Middleware =  (
    @escaping DispatchFunction, @escaping GetState
) -> (@escaping DispatchFunction) -> DispatchFunction
```

Any suggestions?